src/frontend/js/delivery.js

```diff
@@ -7,4 +7,12 @@ $(function() {
         window.location.replace($url+value);
     });
 
+    $('.button.orders').click(function(){
+        $('.button.orders i').addClass('loading');
+        var url = $(this).attr('data-url');
+        $.get(url, function( data ) {
+            window.location.reload();
```

Comment:
@erozqba I have been lazy, I was going to implement exactly that. The rationale behind using an AJAX call is to display a visual hint to the user that the page is loading, as it may be quite some time on production's setup.
How do you propose to reload the DOM with the updated orders list ?